tests/source/issue-4671.rs

```diff
@@ -0,0 +1,11 @@
+struct X<'a>(
+    #[X(X = "_________________________________________________________________________")]
+    pub &'a u32,
+    // ^^
+);
```

Comment:
Could you add a snippet to this file that includes one that starts with the double space too?

Reply:
Would've been a nice to have but not necessary, will go ahead and merge